check_and_format.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+pushd headers
+make
+make format-check
+make symbol-check
+#make symbol-check-extra
+popd
+
+cargo run -- check -r -Vvbomu -d screaming_snake_case -f pascalcase symbols/**.yml
+cargo run -- fmt -r symbols/**.yml
```

Comment:
We're doing a real format here, but with the headers we're only doing a format check. I think we should be consistent and make this a check too:

```suggestion
cargo run --release -- fmt --check -r symbols/*.yml
```

Reply:
Oh! I thought the format-check was both formatting and checking. I’ll instead update it so it format both.